core/src/main/java/org/apache/brooklyn/util/core/internal/ssh/SshTool.java

```diff
@@ -49,7 +50,14 @@
      * These keys are detected from entity/global config and automatically applied to ssh executions. */
     public static final String BROOKLYN_CONFIG_KEY_PREFIX = "brooklyn.ssh.config.";
     
-    public static final ConfigKey<String> PROP_TOOL_CLASS = newStringConfigKey("tool.class", "SshTool implementation to use", null);
+    /**
+     * @deprecated since 0.9.0; use {@link SshMachineLocation#SSH_TOOL_CLASS}
+     */
+    @Deprecated
+    public static final ConfigKey<String> PROP_TOOL_CLASS = newStringConfigKey(
```

Comment:
agree wtih @bostko about pattern where we share the static config key definition; but this particular key name is horrible so +1 to the deprecation.